R/dataframe.R

```diff
@@ -85,9 +85,11 @@ lst_ <- function(xs) {
   names(output) <- character(n)
 
   for (i in seq_len(n)) {
-    output[[i]] <- lazyeval::lazy_eval(xs[[i]], output)
+    output[[i]] <- lazyeval::lazy_eval(xs[[i]], output) %||% list(NULL)
```

Comment:
Would it work if you simply omit the assignment if the result of lazyeval::lazy_eval(xs[[i]], output) is NULL? That might get rid of the postprocessing below, too.